index.html

```diff
@@ -12,6 +12,30 @@
 </head>
 
 <body>
+  <!--Contact info-->
+  <section class="contact-info">
+      <div class="center">
+        <h1>Get In Touch</h1>
```

Comment:
hi boam, <h1> is only used on the top of the page, can you please change it to <h2>?